tests/foreman/api/test_activationkey.py

```diff
@@ -10,7 +9,24 @@
 from robottelo.test import APITestCase
 
 
-@ddt
+def _good_max_content_hosts():
+    """Return a generator yielding valid ``max_content_hosts`` values."""
+    return (gen_integer(*limits) for limits in ((1, 20), (10000, 20000)))
```

Comment:
The outermost parentheses define a generator. I could also use square brackets to define a list, but I prefer generators over lists when possible.

```python
>>> g = (i for i in range(10))
>>> g
<generator object <genexpr> at 0x7fce52fee048>
>>> l = [i for i in range(10)]
>>> l
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```